mate-panel/libmate-panel-applet-private/panel-applet-container.c

```diff
@@ -629,32 +620,33 @@ mate_panel_applet_container_child_set (MatePanelAppletContainer *container,
 {
 	GDBusProxy               *proxy = container->priv->applet_proxy;
 	const AppletPropertyInfo *info;
-	GSimpleAsyncResult       *result;
-
+	GTask                    *task;
 	if (!proxy)
 		return NULL;
 
 	info = mate_panel_applet_container_child_property_get_info (property_name);
 	if (!info) {
-		g_simple_async_report_error_in_idle (G_OBJECT (container),
+		g_task_report_new_error (G_OBJECT (container),
 						     callback, user_data,
+						     NULL,
 						     MATE_PANEL_APPLET_CONTAINER_ERROR,
 						     MATE_PANEL_APPLET_CONTAINER_INVALID_CHILD_PROPERTY,
 						     "%s: Applet has no child property named `%s'",
 						     G_STRLOC, property_name);
 		return NULL;
 	}
 
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_child_set);
+	task = g_task_new (G_OBJECT (container),
+                                    cancellable,
+                                    callback,
+                                    user_data);
+	g_task_set_source_tag (task,mate_panel_applet_container_child_set);
```

Comment:
It looks worse because of missing tabs and wrong alignment. Should be....
```
-	result = g_simple_async_result_new (G_OBJECT (container),
-					    callback,
-					    user_data,
-					    mate_panel_applet_container_child_set);
+	task = g_task_new (G_OBJECT (container),
+			   cancellable,
+			   callback,
+			   user_data);
+	g_task_set_source_tag (task,mate_panel_applet_container_child_set);
```